service/controller/v1/resource_set.go

```diff
@@ -264,8 +259,7 @@ func NewResourceSet(config ResourceSetConfig) (*controller.ResourceSet, error) {
 
 	{
 		c := retryresource.WrapConfig{
-			BackOffFactory: func() backoff.BackOff { return backoff.WithMaxTries(backoff.NewExponentialBackOff(), ResourceRetries) },
-			Logger:         config.Logger,
+			Logger: config.Logger,
```

Comment:
So this backoff factory is only used in v1?

Reply:
Due to the migration to our backoff lib there is no `NewExponentialBackOff` anymore. In fact the `retryresource` in operatorkit defaults to a backoff factory. It is just that we never made use of the defaulting because we anticipated the `v1` package goes away anyway. Now I just had to make it compile so I simply removed it because the defaulting covers it after all.

Reply:
Cool thanks for the explanation.